### PR TITLE
Use default `sync_systems = true` if not specified

### PR DESCRIPTION
**Pull request by digitaltom**
## Description

 Switching the check to run the systems sync from checking the `sync_systems`
value from 'true' to 'not false'. This enables the default behavior to
sync systems on RMTs that kept their config file from a version before systems
syncing was implemented (v2.5.0, Dec 2019)

Drop unneccessary silencing of 'puts' and 'write' calls.

Fixes # (issue)

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [ ] I have verified that my code follows RMT's coding standards with `rubocop`.
- [ ] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [ ] RMT's test coverage remains at 100%.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.

### CODE REVIEW

File: package/obs/rmt-server.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Feb 17 16:49:47 UTC 2022 - Thomas Schmidt <tschmidt@suse.com>
+
+- Fix default behavior to sync systems to SCC ("sync_systems: true" 
+  in /etc/rmt.conf) when the config file is missing the setting
```
Comment (digitaltom) [inline package/obs/rmt-server.changes:5]:
```suggestion
- Enable users with old versions of RMT to sync systems with SCC by default
```

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/238605/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged ece38e716f6e35642f84130b64e1f2d0ae3266e8 into 30cbbd46c28fa4f9fd353478cdd33c24b86e1c81)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/239930/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged f6d76bfab8075811ce89d8518242b1ac41f20534 into 30cbbd46c28fa4f9fd353478cdd33c24b86e1c81)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/240057/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged f6d76bfab8075811ce89d8518242b1ac41f20534 into 70a6550f2244474da8848ab49f0ca2e55dbe1bc4)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/240536/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged f6d76bfab8075811ce89d8518242b1ac41f20534 into 8c4553d761653f359c40c0802e8467e8b414596d)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/240560/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged f6d76bfab8075811ce89d8518242b1ac41f20534 into c1a3c5b7b2eae5343d70f029e1154331ed447e6e)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/240651/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged f6d76bfab8075811ce89d8518242b1ac41f20534 into 9a138413e0dfee3008a477858111cd804752ef25)

**Review by josegomezr — APPROVED** [review]
LGTM :+1:

**Pull request merged**